tensorflow_addons/image/tests/distance_transform_test.py

```diff
@@ -126,5 +126,5 @@ def test_all_zeros():
 def test_all_ones():
     image = tf.ones([10, 10, 1], tf.uint8)
     output = dist_ops.euclidean_dist_transform(image)
-    expected_output = np.full([10, 10, 1], tf.float32.max)
+    expected_output = np.full([10, 10, 1], tf.math.sqrt(tf.float32.max))
```

Comment:
Do you think that we could test the new algo against `scipy.ndimage.distance_transform_edt`?